### ISSUE TITLE
Start position of the player

### ISSUE THREAD

**Issue by bentref** [FelixMo42/hex_game#13]
Right now I've hardcoded the player to start at (-1, 4) because that's a walkable tile on our default map, but I think it would be better to either have either a specific, special start tile (the player's home tile where they build up their base around, perhaps) or to choose a random walkable, harmless tile.

Let me know which of these ideas you prefer and I'll do it

**Comment by FelixMo42** [comment]
It feels like to me this follows from how we end up deciding to generate the world.

**Comment by bentref** [comment]
Yes, absolutely. So in our meeting, world generation should probably be high on the agenda.
